src/test/java/seedu/address/model/tag/TagManagerImplTest.java

```diff
@@ -0,0 +1,110 @@
+package seedu.address.model.tag;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalPersons.ALICE;
+import static seedu.address.testutil.TypicalPersons.BENSON;
+import static seedu.address.testutil.TypicalPersons.FIONA;
+import static seedu.address.testutil.TypicalPersons.GEORGE;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.person.Person;
+import seedu.address.testutil.PersonBuilder;
+
+
+public class TagManagerImplTest {
+
+    public static final Tag TAG_FRIENDS = new Tag("friends");
+    public static final Tag TAG_OWES_MONEY = new Tag("owesMoney");
+    public static final Tag TAG_MODULE = new Tag("CS2103");
+    public static final Tag TAG_NOT_FOUND = new Tag("asdimsad");
+    public static final Tag TAG_SCHOOL = new Tag("NUS");
+    public static final Person BENSON_EDITED = new PersonBuilder(BENSON).withTags("friends", "CS2103", "NUS").build();
+
+    public static TagManagerImpl createEmptyTagManager() {
+        return new TagManagerImpl();
+    }
+
+    /** Contains the tags "friends" (ALICE & BENSON) and "owesMoney" (BENSON) */
```

Comment:
these should be spaced out to three lines

Reply:
As it turns out, private comment Javadocs are not really standardised throughout the code. I think this is not a big issue

Reply:
do you have an example?

Reply:
Ok, as it turns out it's only one instance from the original. The description of the `getFilteredList()` from the `Model` interface. I'll change it